src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/HealthCertifiedPerson/Cells/AdmissionState/AdmissionStateTableViewCell.swift

```diff
@@ -47,6 +47,7 @@ class AdmissionStateTableViewCell: UITableViewCell, UITextViewDelegate, ReuseIde
 
 		faqLinkTextView.attributedText = cellModel.faqLink
 		faqLinkTextView.isHidden = (cellModel.faqLink?.string ?? "").isEmpty
+		faqLinkTextView.accessibilityTraits = .button
 		faqLinkTextView.accessibilityIdentifier = AccessibilityIdentifiers.HealthCertificate.AdmissionState.faq
```

Comment:
Totally makes sense to add the button trait 👍 Could you move it to the `private let faqLinkTextView` configuration as it is not depending on any information from the cellModel? Actually would also make sense for all the accessibility identifiers that we have here, now that I see this. 

And can you also add the trait on the `VaccinationState` cell? I think it would make sense that it is announced in VoiceOver as a button as well.

Reply:
it is fixed now